themes/default/content/docs/pulumi-cloud/esc/providers/aws-secrets.md

```diff
@@ -33,6 +33,31 @@ aws:
           secretId: app-secret
 ```
 
+## Configuring OIDC
+
+To configure OIDC between Pulumi Cloud and AWS, see the [relevant Pulumi documentation](/docs/pulumi-cloud/oidc/aws/). Once you have completed these steps, you can validate that your configuration is working by running either of the following:
+
+* `esc open <your-org>/<your-environment>` command of the [ESC CLI](/docs/esc-cli/)
```

Comment:
I _think_ the guidance is that we always use "Pulumi ESC" and never just "ESC". ([Here's a thread](https://pulumi.slack.com/archives/C05G9TV06MB/p1696898160079599) for reference, but feel free to confirm if you think this has changed.)

```suggestion
* `esc open <your-org>/<your-environment>` command of the [Pulumi ESC CLI](/docs/esc-cli/)
```